### ISSUE TITLE
ERROR TypeError: this.dropdown.hostElement.nativeElement.firstChild.getBoundingClientRect is not a function

### ISSUE THREAD

**Issue by igniens** [basvandenberg/ng-select#296]
template:
```
                  <form class="form-material">
                      <div class="form-group form-primary">
                        <ng-select [ngClass]="'ng-select'" [options]="countries"  [ngModelOptions]="{standalone: true}" [allowClear]="true" [(ngModel)]="selectedCountry">
                          <ng-template #optionTemplate let-option="option">
                            <div class="famfamfam-flags {{option?.value.toLowerCase()}}"></div>
                            {{option?.label}}
                          </ng-template>
                        </ng-select>
                          <span class="form-bar"></span>
                      </div>
                  </form>
```
error:
ERROR TypeError: this.dropdown.hostElement.nativeElement.firstChild.getBoundingClientRect is not a function
    at SelectComponent.updatePosition (ng-select.es5.js:1807)
    at eval (ng-select.es5.js:1634)
    at ZoneDelegate.invokeTask (zone.js:421)
    at Object.onInvokeTask (core.js:4740)
    at ZoneDelegate.invokeTask (zone.js:420)
    at Zone.runTask (zone.js:188)
    at ZoneTask.invokeTask (zone.js:496)
    at ZoneTask.invoke (zone.js:485)
    at timer (zone.js:2025)
happens when you open select

**Comment by basvandenberg** [comment]
This looks very similar to issue #287 and #289, which has been fixed in version 1.0.0-rc.5. Just to be sure which version are you using? If you are using rc.4, then pleas update to rc.5 to resolve this issue.

**Comment by basvandenberg** [comment]
Since there has been no response, I assume that this has been resolved by upgrading to a newer version.
